hedera-node/hedera-app/src/main/java/com/hedera/node/app/grpc/impl/usage/UserAgentType.java

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: Apache-2.0
+package com.hedera.node.app.grpc.impl.usage;
+
+import static java.util.Objects.requireNonNull;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Enum containing the known types of SDKs, along with values for cases where they SDK type is not known or not specified.
+ */
+public enum UserAgentType {
+    HIERO_SDK_CPP("HieroSdkCpp", true, "hiero-sdk-cpp"),
+    HIERO_SDK_GO("HieroSdkGo", true, "hiero-sdk-go"),
+    HIERO_SDK_JAVA("HieroSdkJava", true, "hiero-sdk-java"),
+    HIERO_SDK_JS("HieroSdkJs", true, "hiero-sdk-js"),
+    HIERO_SDK_PYTHON("HieroSdkPython", true, "hiero-sdk-python"),
+    HIERO_SDK_RUST("HieroSdkRust", true, "hiero-sdk-rust"),
+    HIERO_SDK_SWIFT("HieroSdkSwift", true, "hiero-sdk-swift"),
+    UNSPECIFIED("Unspecified", false),
```

Comment:
Q: Would direct gRPC calls from non-SDKs fall into this bucket or `UNKNOWN` or sometimes both?
Would be good to know that set of users separately also.